proposals/simd/NewOpcodes.md

```diff
@@ -108,6 +108,7 @@
 | i8x16.max_u          | 0x79   | i16x8.max_u              | 0x99   | i32x4.max_u              | 0xb9   | ----        | 0xd9   |
 | ----------------     | 0x7a   | ----------------         | 0x9a   | i32x4.dot_i16x8_s        | 0xba   | ----        | 0xda   |
 | i8x16.avgr_u         | 0x7b   | i16x8.avgr_u             | 0x9b   | ---- avgr_u ----         | 0xbb   | ----        | 0xdb   |
+| ----                 | 0x7c   | i16x8.q15mulr_sat_s      | 0x9c   | ----                     | 0xbc   | ----        | 0xdc   |
```

Comment:
Can you add TBD here, too, so that the two opcode files are consistent?

Reply:
Reverted the modification of this file